python/fileaway.py

```diff
@@ -72,19 +72,21 @@
 #  -Handles improper commands
 # Version 1.0.2 release - Jun 15, 2011
 #  -Added alternative for xset users (credit: sherpa9 at irc.freenode.net)
+# Version 1.0.3 release - Dec 19, 2019
+#  -Adapt for python3
 
```

Comment:
To use the print function in Python 2, you should add this import:

```python
from __future__ import print_function
```

Reply:
added, tnx